ApplicationCode/ProjectDataModel/CorrelationPlots/RimAbstractCorrelationPlot.h

```diff
@@ -60,9 +60,10 @@ class RimAbstractCorrelationPlot : public RimPlot
     int axisTitleFontSize() const;
     int axisValueFontSize() const;
 
-    void setLabelFontSize( caf::FontTools::RelativeSize fontSize );
-    void setAxisTitleFontSize( caf::FontTools::RelativeSize fontSize );
-    void setAxisValueFontSize( caf::FontTools::RelativeSize fontSize );
+    void             setLabelFontSize( caf::FontTools::RelativeSize fontSize );
+    void             setAxisTitleFontSize( caf::FontTools::RelativeSize fontSize );
+    void             setAxisValueFontSize( caf::FontTools::RelativeSize fontSize );
+    std::set<time_t> allAvailableTimeSteps();
```

Comment:
Should probably be const?

Reply:
Unfortunately no. It uses getOrCreateCurveDefAnalyzer which creates the analyzer on demand, thus changing the object.